src/App.tsx

```diff
@@ -21,14 +68,21 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && (
+                <Loader />
+              )}
+
+              {todos.length && (
```

Comment:
```suggestion
              {!!todos.length && (
```
![image](https://user-images.githubusercontent.com/97547824/223772828-ed63b695-d7c1-4e42-bc25-f4850cd94575.png)